specification/metrics/pricingquantity.md

```diff
@@ -0,0 +1,31 @@
+# Pricing Quantity
+
+The Pricing Quantity represents the volume of a given resource and/or service used or purchased based on the [Pricing Measure](#pricinmeasure). Distinct from [Usage Quantity](#usagequantity) (complementary to [Usage Unit](#usageunit)), it focuses on pricing and cost, not resource and service consumption.
+
+PricingQuantity MUST be present in the billing data. This column MUST be a numeric value of type Decimal. The value MAY be negative in cases where [ChargeCategory](#chargecategory) is "Adjustment". This column MUST NOT contain null values where unit prices are not null. When unit prices are not null, multiplying PricingQuantity by a unit price MUST produce a result equal to the corresponding cost metric.
```

Comment:
```suggestion
PricingQuantity MUST be present in the billing data. This column MUST be a numeric value of type Decimal. The value MAY be negative in cases where [ChargeCategory](#chargecategory) is "Adjustment" and MUST be negative in cases where [ChargeSubcategory](#chargesubcategory) is "Credit". This column MUST NOT contain null values where unit prices are not null. When unit prices are not null, multiplying PricingQuantity by a unit price MUST produce a result equal to the corresponding cost metric.
```

Reply:
Not sure if we should impose negative values in case of credit.

Reply:
I would think in the case of ad-hoc credits, we would not want to do this. Putting a negative quantity implies that we have a rate associated with the charge? which we don't since there's no SKU? Interested to understand if I'm thinking of this wrong.

Reply:
Agreed, it's strange. As currently defined, enforcing a negative `pricingQuantity` for a credit would force a non-null `pricingMeasure`. Any quantity `pricingMeasure` strengthens the implication that there are rates/measures associated with all credits. Vote to not include.

Reply:
Thanks for your feedback! Closing the conversation